includes/TripalImporter/BibtexImporter.inc

```diff
@@ -0,0 +1,108 @@
+<?php
+
+class BibtexImporter extends TripalImporter {
+
+  /**
+   * The name of this loader.  This name will be presented to the site
+   * user.
+   */
+  public static $name = 'Chado bibtex Loader';
+
+  /**
+   * The machine name for this loader. This name will be used to construct
+   * the URL for the loader.
+   */
+  public static $machine_name = 'chado_bibtex_loader';
+
+  /**
+   * A brief description for this loader.  This description will be
+   * presented to the site user.
+   */
+  public static $description = 'Load bibtex formatted publications into Chado';
+
+  /**
+   * An array containing the extensions of allowed file types.
+   */
+  public static $file_types = [
+    '.txt',
+    '.tex',
+    'bib',
+  ];
+
+  /**
+   * Allow, but dont require, an analysis.
+   */
+  public static $use_analysis = TRUE;
+
+  public static $require_analysis = FALSE;
+
+  /**
+   * Provides information to the user about the file upload.  Typically this
+   * may include a description of the file types allowed.
+   */
+  public static $upload_description = 'Please provide the bibtex file. The file must have a .txt, .tex, or .bib extension.';
+
+  /**
+   * The title that should appear above the file upload section.
+   */
+  public static $upload_title = 'Bibtex File Upload';
+
+  /**
+   * Text that should appear on the button at the bottom of the importer
+   * form.
+   */
+  public static $button_text = 'Import bibtex file';
+
+
+  /**
+   * Indicates the methods that the file uploader will support.
+   */
+  public static $methods = [
+    // Allow the user to upload a file to the server.
+    'file_upload' => TRUE,
+    // Allow the user to provide the path on the Tripal server for the file.
+    'file_local' => TRUE,
+    // Allow the user to provide a remote URL for the file.
+    'file_remote' => TRUE,
+  ];
+
+  /**
+   * @see TripalImporter::form()
+   */
+  public function form($form, &$form_state) {
+
+
+    $form['instructions'] = [
```

Comment:
Add `'#weight' => -100` to make sure the instructions are at the top of the page. Currently the form looks a bit weird:
![screen shot 2018-06-06 at 3 54 02 pm](https://user-images.githubusercontent.com/1566301/41067182-e28aaaa2-69a1-11e8-948b-c2c467d48d43.png)

Reply:
cool, didnt know i could do that.